bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -31,6 +31,7 @@ class Neuron(CMakePackage):
     patch("patch-v800-cmake-nvhpc.patch", when="@8.0.0%nvhpc^cmake@3.20:")
 
     version("develop", branch="master")
+    version("8.2.2a", commit="eb19ae0")
```

Comment:
Just copying the slack discussing here:  when we will have the actual 8.2 release then this version name will be problematic:

```python
>>> Version("8.2.2") > Version("8.2.2a")
False
```
i.e. such version naming doesn't work well with Spack. Some alternatives are like:

```python
>>> Version("9.b1") > Version("9.a1")
True
>>> Version("9.a2") > Version("9.a1")
True
>>> Version("9.0") > Version("9.a1")
True
```